<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="launchBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>200</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>START AI CAMERA</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_heading">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>721</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; font-weight:700;&quot;&gt;DROWSINESS DETECTION APPLICATION&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_creators">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>470</y>
      <width>56</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>created by</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name1">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>500</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Anupam Verma</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_name2">
    <property name="geometry">
     <rect>
      <x>650</x>
      <y>520</y>
      <width>82</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Jatin Mishra</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_status">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>420</y>
      <width>291</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;Status&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBtn">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>300</y>
      <width>291</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Segoe UI Black</family>
      <pointsize>18</pointsize>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>EXIT APP</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
